ui/app/js/app.js

```diff
@@ -406,21 +406,21 @@ angular.module('am').config(
   function($routeProvider) {
     $routeProvider.
     when('/alerts', {
-      templateUrl: '/app/partials/alerts.html',
+      templateUrl: 'app/partials/alerts.html',
       controller: 'AlertsCtrl',
       reloadOnSearch: false
     }).
     when('/silences', {
-      templateUrl: '/app/partials/silences.html',
+      templateUrl: 'app/partials/silences.html',
       controller: 'SilencesCtrl',
       reloadOnSearch: false
     }).
     when('/status', {
-      templateUrl: '/app/partials/status.html',
+      templateUrl: 'app/partials/status.html',
       controller: 'StatusCtrl'
     }).
     otherwise({
       redirectTo: '/alerts'
```

Comment:
As far as I can tell, this all works fine for me. Have you tested it? E.g. I run:

```
./alertmanager -web.external-url=http://localhost:9093/foo/bar
```

Going to `http://localhost:9093/foo/bar` correctly redirects me to `http://localhost:9093/foo/bar/#/alerts`.

Note that this is an angular route name that appears in the hash part of the URL, and is thus not part of the normal path.

Or am I misunderstanding something?